README.md

```diff
@@ -148,6 +149,22 @@ Example:
 
 ### Widgets list
 
+- `V-Card` Display a v-card for a given user, with some generic, and public, information. Usable in unlogged and logged mode.
+    - `name`: `vcard` *(mandatory)*
+    - `options`:
+        - `userProfile`: {id: [user ID]} *(mandatory)*
+        - `bodyHeight`: "[height in pixel or %]" *(optional)* _mixte_, If you want to force a specifc height for the bdoy part of the widget, indicate it here. It's taking the same type of value of the CSS width parameter: a specific width in pixel (`100px`) or in percentage (`50%`).
```

Comment:
Same here, two instances of "width" that should be "height" if I'm not mistaken.